contract/src/auctionItems.js

```diff
@@ -66,6 +66,25 @@ const start = (zcf) => {
     updater: availableItemsUpdater,
   } = makeNotifierKit();
 
+  const addToUserSaleHistory = (cardAmount) => {
+    try {
+      const validatedAmount = AmountMath.coerce(brands.Items, cardAmount);
```

Comment:
What does coerce do?

Reply:
It basically throws an error if the card Amount is not of the expected brand.And if it is of the right brand it returns validated amount

Reply:
It basically throws an error if the card Amount is not of the expected brand.And if it is of the right brand it returns validated amount